doc/source/reference/c-api.coremath.rst

```diff
@@ -150,31 +150,31 @@ Those can be useful for precise floating point comparison.
 
     .. versionadded:: 1.4.0
 
-.. c:function:: void npy_set_floatstatus_divbyzero()
+.. c:function:: void npy_set_floatstatus_divbyzero(void*)
```

Comment:
do we need barriers on the set functions?
reordering becomes interesting when reading the status but setting doesn't really matter as it has no influence on future instructions.